.golangci.yml

```diff
@@ -39,6 +39,15 @@ linters:
   - whitespace
 
 linters-settings:
+  godot:
+    #   declarations - for top level declaration comments (default);
+    #   toplevel     - for top level comments;
+    #   all          - for all comments.
+    scope: toplevel
+    exclude:
+    - '^ \+.*'
+    - '^ ANCHOR.*'
+    capital: true
```

Comment:
Do we need this? It seems flaky imo and lots of friction. The benefit of having a capital letter after a dot is small compared to a smoother new contributor experience

Reply:
I would drop capitalization as stated in: https://github.com/kubernetes-sigs/cluster-api/pull/5591#pullrequestreview-798648322

Just wanted to wait for additional opinions.

Reply:
Done